integreat_cms/matomo_api/apps.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import json
+import requests
+
+from django.apps import apps, AppConfig
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class MatomoApiManagerConfig(AppConfig):
+    """
+    MatomoApiManager config inheriting the django AppConfig
+    """
+
+    name = "integreat_cms.matomo_api"
+    api_available = False
+
+    def ready(self):
```

Comment:
Since the ready function is mostly just a try/except to me it doesn't really make sense to declutter the function. Or does someone of you have an idea on how to reduce the Cognitive Complexity? :)